source/user-manual/ruleset/ruleset-xml-syntax/rules.rst

```diff
@@ -201,6 +205,17 @@ Any IP address or CIDR block to be compared to an IP decoded as dstip. Use "!" t
 | **Allowed values** | Any dstip |
 +--------------------+-----------+
 
+data
+^^^^^^^^^^
+
+Any string that is decoded into the data field.
+
++--------------------+-------------+
+| **Default Value**  | n/a         |
++--------------------+-------------+
+| **Allowed values** | Any string. |
```

Comment:
Looking at the code of https://github.com/wazuh/wazuh/pull/3591, `data` can be an `SRegex` as well.